### ISSUE TITLE
Implementing Bag of Tricks for Node Classification with Graph Neural Networks

### ISSUE THREAD

**Issue by mattjhayes3** [pyg-team/pytorch_geometric#9831]
### 🚀 The feature, motivation and pitch

Hello PyG team,
@chriskynguyen, @liuvince, and I are CS224W students and have been working on implementing tricks from https://arxiv.org/abs/2103.13355.  

We hope PyG users will find the following helpful to “level the playing field” for future research and squeeze extra performance out for production models.  We’ll be linking separate pull requests for each of the tricks they discuss that are not already implemented:
* “Label Usage”
* GAT with Symmetric Normalized Adjacency Matrix
* “Non-interactive” GAT
* Log-$\epsilon$ loss

We would much appreciate your consideration and review of any of these.  
Additional descriptions and pull requests to come shortly.

Thank you!

### Alternatives

To be discussed in individual PRs.
